src/test/java/br/fatec/bd4/RegistroControllerTest.java

```diff
@@ -0,0 +1,38 @@
+package br.fatec.bd4;
+
```

Comment:
André, about the test class, it is needed to think how we can code the test without our cloud database, because you are requiring data that in this moment are posted on DB, but if I delete all data, the test won't execute, right?

Reply:
You're right. Let's talk in private to make these tests better.